packages/components/psammead-paragraph/package.json

```
@@ -26,6 +26,7 @@
     "@bbc/psammead-styles": "^0.3.2"
   },
   "devDependencies": {
+    "@bbc/psammead-inline-link": "1.0.0",
```

```suggestion
    "@bbc/psammead-inline-link": "1.1.0",
```

Bump to `1.1.0` after https://github.com/bbc/psammead/pull/707 is merged and deployed. Also run `npm install`